docs/platforms/java/common/enriching-events/scopes/index.mdx

```diff
@@ -91,6 +156,7 @@ caught, and all errors that occur will be silently ignored and **not** reported.
 
 </Alert>
 
+
 ## Kotlin Coroutines
```

Comment:
Should Coroutines also be mentioned on the Android side?

Reply:
Not sure, can ask Android folks

Reply:
Created https://github.com/getsentry/sentry-java/issues/4073